htdocs/translate.php

```diff
@@ -88,136 +88,149 @@
-
-                                                                                    $action = 'listnew';
-
-                                                                                    $trans = sql(
-                                                                                        "SELECT DISTINCT `sys_trans`.`id`, `sys_trans`.`text` FROM `sys_trans` LEFT JOIN `sys_trans_text` ON `sys_trans`.`id`=`sys_trans_text`.`trans_id` AND `sys_trans_text`.`lang`='&1' LEFT JOIN `sys_trans_ref` ON `sys_trans`.`id`=`sys_trans_ref`.`trans_id` WHERE (ISNULL(`sys_trans_text`.`trans_id`) OR `sys_trans_text`.`text`='') ORDER BY `sys_trans`.`id` DESC",
-                                                                                        $translang
-                                                                                    );
-                                                                                    $tpl->assign_rs('trans', $trans);
-                                                                                    sql_free_result($trans);
-                                                                                }
-                                                                            }
-                                                                        }
-                                                                    }
-                                                                }
-                                                            }
-                                                        }
-                                                    }
-                                                }
-                                            }
-                                        }
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+        break;
+
+    case 'resetids':
+        resetIds();
+        break;
+
+    case 'clearcache':
+        clearCache();
+        break;
+
+    case 'export':
+        export();
+        break;
+
+    case 'xmlexport':
+        xmlexport();
+        break;
+
+    case 'xmlimport':
+        break;
+
+    case 'xmlimport2':
+        xmlimport2();
+        break;
+
+    case 'xmlimport3':
+        xmlimport3();
+        break;
+
+    case 'textexportnew':
+        textexport($translang, false);
+        break;
+
+    case 'textexportall':
+        textexport($translang, true);
+        break;
+
+    case 'textimport':
+        break;
+
+    case 'textimport2':
+        textimport($translang);
+        break;
+
+    case 'edit':
+        if (!$access->mayTranslate($translang)) {
+            $tpl->error(ERROR_NO_ACCESS);
         }
-    }
+        edit();
+        break;
+
+    case 'copy_en':
+        copy_english_texts();
+        break;
+
+    case 'listfaults':
+        $trans = sql(
+            "SELECT
```

Comment:
There was no note, and I think we should keep this consistent and enclose all SQL statements in double quotes.